Modules/overlapped.c

```diff
@@ -1904,41 +1897,67 @@ static PyMethodDef overlapped_functions[] = {
     {NULL}
 };
 
-static struct PyModuleDef overlapped_module = {
-    PyModuleDef_HEAD_INIT,
-    "_overlapped",
-    NULL,
-    -1,
-    overlapped_functions,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
+static int
+overlapped_traverse(PyObject *module, visitproc visit, void *arg)
+{
+    OverlappedState *state = overlapped_get_state(module);
+    Py_VISIT(state->overlapped_type);
+    return 0;
+}
 
-#define WINAPI_CONSTANT(fmt, con) \
-    PyDict_SetItemString(d, #con, Py_BuildValue(fmt, con))
+static int
+overlapped_clear(PyObject *module)
+{
+    OverlappedState *state = overlapped_get_state(module);
+    Py_CLEAR(state->overlapped_type);
+    return 0;
+}
 
-PyMODINIT_FUNC
-PyInit__overlapped(void)
+static void
+overlapped_free(void *module)
 {
-    PyObject *m, *d;
+    overlapped_clear((PyObject *)module);
+}
 
+#define WINAPI_CONSTANT(fmt, con) \
```

Comment:
I think that this can be converted into `PyModule_AddObject` instead of using PyModule_GetDict

Reply:
@corona10 You're right.  Is that preferable to PyModule_GetDict? If so why?

Reply:
https://docs.python.org/3/c-api/module.html?highlight=pymodule_getdict#c.PyModule_GetDict

```
It is recommended extensions use other PyModule_*() and PyObject_*() functions rather than directly manipulate a module’s __dict__.
```

Reply:
@corona10 I'm trying that now.  So far it is not looking good - somehow the values are all wrong.  For example when I switch to `PyModule_AddObject`, `overlapped.__dict__["ERROR_IO_PENDING"]` is 6160 instead of 997.  I can't figure out why..